execution_engine/src/storage/store/mod.rs

```diff
@@ -31,6 +38,8 @@ pub trait Store<K, V> {
         }
     }
 
+    /// `put` puts a `value` into the store at `key` within a transaction, potentially returning an
```

Comment:
```suggestion
    /// Puts a `value` into the store at `key` within a transaction, potentially returning an
```
Probably also worth stating what happens if the value already exists.